modules/apps/transfer/keeper/msg_server_test.go

```diff
@@ -191,31 +121,31 @@ func (suite *KeeperTestSuite) TestMsgTransfer() {
 			path = ibctesting.NewTransferPath(suite.chainA, suite.chainB)
 			path.Setup()
 
-			coin1 = sdk.NewCoin(sdk.DefaultBondDenom, sdkmath.NewInt(100))
 			msg = types.NewMsgTransfer(
 				path.EndpointA.ChannelConfig.PortID,
 				path.EndpointA.ChannelID,
-				sdk.NewCoins(coin1),
+				testCoins,
 				suite.chainA.SenderAccount.GetAddress().String(),
 				suite.chainB.SenderAccount.GetAddress().String(),
 				suite.chainB.GetTimeoutHeight(), 0, // only use timeout height
 				"memo",
 			)
 
+			// send some coins of the second denom from bank module to the sender account as well
+			err := suite.chainA.GetSimApp().BankKeeper.MintCoins(suite.chainA.GetContext(), types.ModuleName, sdk.NewCoins(coin2))
+			suite.Require().NoError(err)
+			err = suite.chainA.GetSimApp().BankKeeper.SendCoinsFromModuleToAccount(suite.chainA.GetContext(), types.ModuleName, suite.chainA.SenderAccount.GetAddress(), sdk.NewCoins(coin2))
+			suite.Require().NoError(err)
```

Comment:
is it possible for us to not need to do this based on the `TestCoins` value? Seems like we'll need this boilerplate everywhere we want to send multiple denoms